public/app/features/transformers/docs/content.ts

```diff
@@ -0,0 +1,1024 @@
+/*
+  NOTE: This file is used to generate the transformation docs markdown content. If you change/update the content here,
+  please then rebuild the markdown by doing the following:
+
+  $ cd /docs (from the root of the repository)
+  $ make sources/panels-visualizations/query-transform-data/transform-data/index.md
+  $ make docs
+
+  Browse to http://localhost:3003/docs/grafana/latest/panels-visualizations/query-transform-data/transform-data/
+
+  Refer to ./docs/README.md for more information about building docs. 
+*/
+
+interface Link {
+  title: string;
+  url: string;
+}
+export interface TransformationInfo {
+  name: string;
+  helperDocs: string;
+  links?: Link[];
+}
+
+export const transformationDocsContent: Record<string, TransformationInfo> = {
+  calculateField: {
+    name: 'Add field from calculation',
+    helperDocs: `
+  Use this transformation to add a new field calculated from two other fields. Each transformation allows you to add one new field.
```

Comment:
In the readme can we add what/how the UI pulls content from this file? Is it just the first sentence? Is it more? I ask because I'm not sure what shows up in the UI on something like line 50, the whole line?
```suggestion
  Use this transformation to add a new field calculated from two other fields. Each transformation allows you to add one new field.
```

Reply:
It's the whole `helperDocs` chuck, displayed in a UI model. This will be true for all `helperDocs` pieces. It does feel like a lot of content when just the string is viewed in the code, but in the helper modal it looks a lot more reasonable and natural.